wear/build.gradle

```diff
@@ -117,6 +117,7 @@ dependencies {
     implementation project(':modules:services:preferences')
     implementation project(':modules:services:repositories')
     implementation project(':modules:services:servers')
+    implementation project(':modules:services:sharedtest')
```

Comment:
I think this can also be `testImplementation` instead of `implementation` since it is only used for tests. If you wanted to also make this change in the other places where we depend on the `sharedtest` module as well, that would be great.